examples/js/loaders/gltf/glTFLoader.js

```diff
@@ -826,6 +826,8 @@ THREE.glTFLoader.prototype.load = function( url, callback ) {
 									var opacity =  USE_A_ONE ? value : (1.0 - value);
 									uvalue = opacity;
 									params.transparent = true;
+								} else if (value) {
```

Comment:
I wanted to check the `value is not falsy` since I had no idea what kind of value come. Sorry, I realize it's a little lazy reason now.
And if zero comes to the `value`, the conditional expression is not correct.
Then, How about removing it daringly.
```javascript
} else {
    uvalue = value;
}
```